src/catchup/CatchupWork.cpp

```diff
@@ -25,15 +25,15 @@ namespace stellar
 
 CatchupWork::CatchupWork(Application& app, WorkParent& parent,
                          CatchupConfiguration catchupConfiguration,
-                         bool manualCatchup, ProgressHandler progressHandler,
-                         size_t maxRetries)
+                         ProgressHandler progressHandler, size_t maxRetries,
+                         optional<Hash> trustedScpHash)
```

Comment:
maxRetries is last argument for constructors of all Work items, could you please leave it last here as well for consistency?